docs/contribute/contribute.md

```diff
@@ -0,0 +1,62 @@
+# Contributing to Amber
+This is a simple but exhaustive guide to get you started on contributing to amber.
+
+## Contributing guidelines
+
+Before you dig into Amber, you should know a few things before you contribute.
+
+Any code change is submitted [through a PR](https://github.com/amber-lang/Amber/pulls), which is then approved by at least 2 maintainers.
+
+The way we talk on github is not the same as we would talk in person. When on github, always get straight to the point and be critical.
```

Comment:
Should probably capitalise "GitHub" as well.

Reply:
You still have an uncapitalised "github". Suggest search and replace this throughout the project.